pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/OperatorStats.java

```diff
@@ -61,11 +61,11 @@ public void recordRow(int numBlock, int numRows) {
   }
 
   public void recordExecutionStats(Map<String, String> executionStats) {
-    _statsAggregator.aggregate(executionStats);
+    _executionStats = executionStats;
```

Comment:
could we think of a way to incorporate `_numBlock` and `_numRows` in the executionStats?

Reply:
Makes sense. I was thinking about that as well.